lib/beaker/host/windows/exec.rb

```diff
@@ -5,7 +5,7 @@ def reboot
     exec(Beaker::Command.new('shutdown /r /t 0 /d p:4:1 /c "Beaker::Host reboot command issued"'), :expect_connection_failure => true)
   end
 
-  ABS_CMD = 'c:\\\\windows\\\\system32\\\\cmd.exe'
+  ABS_CMD = "/cygdrive/c/WINDOWS/system32/cmd.exe"
```

Comment:
This part surprises me, cmd.exe is very sensitive to forward slashes in the command line, as it dumbly thinks they are command arguments. It maybe that cygwin coverts that to backslashes before it invokes cmd.exe, but in any case, it seems risky, and unrelated to the mac issue. Maybe separate this out until we can verify windows acceptance is not affected? /cc @Iristyle